src/components/Learningpath/LearningpathEmbed.tsx

```diff
@@ -183,7 +183,7 @@ const LearningpathEmbed = ({ learningpathStep, skipToContentId, subjectId, bread
   const stepArticle = learningpathStepResource?.article;
 
   if (!stepArticle) {
-    return <ErrorPage />;
+    return <DefaultErrorMessage />;
```

Comment:
Her tror jeg det finnes et tilfelle der denne faktisk må ha en skip-ID. Kunne du lagt inn en TODO, så kan jeg se på det etter at vi tar denne inn? Må fikse opp i litt andre greier som ikke er relevant for denne PR'en for å få det til.